dbt_project.yml

```diff
@@ -25,7 +25,7 @@ clean-targets:
 vars:
   snowplow_web:
     # Sources
-    # snowplow__atomic_schema: 'atomic_data_sample' # Only set if not using 'atomic' schema for Snowplow events data
+    snowplow__atomic_schema: 'rt_pipeline_prod1' # Only set if not using 'atomic' schema for Snowplow events data
```

Comment:
Assume this was a local change you forgot to remove before committing, we all do it 😅

Reply:
:sweat_smile: